docs/source/metadata.rst

```diff
@@ -8,29 +12,67 @@ or an e-mail with a given subject line.
+| ``last_modified``           | Last Modified Date                                       |                                                                                                                                                                                                                                                                                             |
++-----------------------------+----------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ``filetype``                | File Type                                                |                                                                                                                                                                                                                                                                                             |
++-----------------------------+----------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ``type``                    | Element Type                                             | Categorizes elements into types such as Title, NarrativeText. Not a metadata field                                                                                                                                                                                                          |
++-----------------------------+----------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ``coordinates``             | XY Bounding Box Coordinates                              |                                                                                                                                                                                                                                                                                             |
```

Comment:
Include in-doc link to further down the page where coordinates are explained in more detail?